azext_hanaonazure/modules_sdk/operations/hana_instances_operations.py

```diff
@@ -241,6 +241,193 @@ def get(
+            raise models.ErrorResponseException(self._deserialize, response)
+
+        deserialized = None
+
+        if response.status_code == 200:
+            deserialized = self._deserialize('HanaInstance', response)
+        if response.status_code == 201:
+            deserialized = self._deserialize('HanaInstance', response)
+
+        if raw:
+            client_raw_response = ClientRawResponse(deserialized, response)
+            return client_raw_response
+
+        return deserialized
+
+    def create(
+            self, resource_group_name, hana_instance_name, hana_instance_parameter, custom_headers=None, raw=False, polling=True, **operation_config):
+        """Creates a SAP HANA instance.
+
+        Creates a SAP HANA instance for the specified subscription, resource
+        group, and instance name.
+
+        :param resource_group_name: Name of the resource group.
+        :type resource_group_name: str
+        :param hana_instance_name: Name of the SAP HANA on Azure instance.
+        :type hana_instance_name: str
+        :param hana_instance_parameter: Request body representing a
+         HanaInstance
+        :type hana_instance_parameter:
+         ~azure.mgmt.hanaonazure.models.HanaInstance
+        :param dict custom_headers: headers that will be added to the request
+        :param bool raw: The poller return type is ClientRawResponse, the
+         direct response alongside the deserialized response
+        :param polling: True for ARMPolling, False for no polling, or a
+         polling object for personal polling strategy
+        :return: An instance of LROPoller that returns HanaInstance or
+         ClientRawResponse<HanaInstance> if raw==True
+        :rtype:
+         ~msrestazure.azure_operation.AzureOperationPoller[~azure.mgmt.hanaonazure.models.HanaInstance]
+         or
+         ~msrestazure.azure_operation.AzureOperationPoller[~msrest.pipeline.ClientRawResponse[~azure.mgmt.hanaonazure.models.HanaInstance]]
+        :raises:
+         :class:`ErrorResponseException<azure.mgmt.hanaonazure.models.ErrorResponseException>`
+        """
+        raw_result = self._create_initial(
+            resource_group_name=resource_group_name,
+            hana_instance_name=hana_instance_name,
+            hana_instance_parameter=hana_instance_parameter,
+            custom_headers=custom_headers,
+            raw=True,
+            **operation_config
+        )
+
+        def get_long_running_output(response):
+            deserialized = self._deserialize('HanaInstance', response)
+
+            if raw:
+                client_raw_response = ClientRawResponse(deserialized, response)
+                return client_raw_response
+
+            return deserialized
+
+        lro_delay = operation_config.get(
+            'long_running_operation_timeout',
+            self.config.long_running_operation_timeout)
+        if polling is True: polling_method = ARMPolling(lro_delay, **operation_config)
```

Comment:
formatting for conditional statements look unconventional. What tool are you using for formatting

Reply:
I couldn't tell you what formatting tool is used; this comes from https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/hanaonazure/azure-mgmt-hanaonazure/azure/mgmt/hanaonazure

Everything under azext_hanaonazure/modules_sdk/* is auto-generated, not written by me